helm-charts/splunk-otel-collector/templates/config/_otel-collector.tpl

```diff
@@ -239,7 +239,7 @@ service:
         {{- end }}
     {{- end }}
 
-    {{- if or (eq (include "splunk-otel-collector.splunkO11yEnabled" $) "true") (eq (include "splunk-otel-collector.platformMetricsEnabled" $) "true") }}
+    {{- if and (or (eq (include "splunk-otel-collector.splunkO11yEnabled" $) "true") (eq (include "splunk-otel-collector.platformMetricsEnabled" $) "true")) (or (eq (include "splunk-otel-collector.splunkO11yEnabled" .) "true") (eq (include "splunk-otel-collector.platformMetricsEnabled" $) "true")) }}
```

Comment:
Why is this change needed? looks like both and argument are the same, no?

Reply:
Oh they are the same now. I missed that. I will fix.

Reply:
see https://github.com/signalfx/splunk-otel-collector-chart/pull/544